vita3k/config/src/config.cpp

```diff
@@ -118,12 +118,12 @@ ExitCode serialize_config(Config &cfg, const fs::path &output_path) {
 ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths) {
     // Always generate the default configuration file
     Config command_line{};
-    serialize_config(command_line, root_paths.get_base_path() / "data/config/default.yml");
-    // Load base path configuration by default; otherwise, move the default to the base path
-    if (fs::exists(check_path(root_paths.get_base_path())))
-        parse(cfg, root_paths.get_base_path(), root_paths.get_base_path());
+    serialize_config(command_line, root_paths.get_pref_path() / "data/config/default.yml");
```

Comment:
for here, depending of your code, data config is inside shared or base path
pref path have nothing, is empty

Reply:
You are right. This file should go to cache_path

Reply:
ah, macdu have more said shared, i have thinks so more config_path/default/

Reply:
Fixed.